SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -69,6 +69,12 @@ Worker_EntityId USpatialPackageMapClient::AllocateEntityIdAndResolveActor(AActor
 	check(Actor);
 	checkf(bIsServer, TEXT("Tried to allocate an Entity ID on the client, this shouldn't happen."));
 
+	if (!IsEntityPoolReady())
+	{
+		UE_LOG(LogSpatialPackageMap, Error, TEXT("EntityPool must be ready when resolving an Actor: %s"), *Actor->GetName());
```

Comment:
Is this an error? Couldn't this just happen in unfavourable initial level loading conditions?

Reply:
This happens on client. As we've discussed, the execution flow shouldn't even get to this stage. I added more checks to terminate the execution earlier.

Reply:
There's a check above for running on server, perhaps that should be extended to return, rather then the inferred logic that the entity pool is setup.